Resources/Maps/deaddrop.yml

```diff
@@ -251,6 +289,27 @@ entities:
     - pos: 1.5,1.5
       parent: 1
       type: Transform
+- proto: DebugAPC
```

Comment:
DebugAPC... ehhh...
Are you sure?

Reply:
was a mistake but also I cant find what makes the debug apc so special anyhow. the 200kw one is way stronger anyhow. But yea, ill make sure it gets changed

Reply:
When you break it you get normal apc, not an issue.